scripts/static-vdis

```diff
@@ -220,12 +220,28 @@ def doexec(args, inputtext=None):
     rc = proc.returncode
     return (rc,stdout,stderr)
 
+
+def connect_smapiv1_nbd(params_nbd):
+    # Format "nbd:unix:<socket path>:exportname=<export name>"
```

Comment:
Does `parse_nbd_uri` not do the right thing?

Reply:
You're correct it does exactly what's required, I just didn't spot it in the code.